### ISSUE TITLE
Improve Recovery Key Metrics

### ISSUE THREAD

**Issue by davismtl** [mozilla/fxa#2375]
- Track an event when someone generates and downloads their recovery key
- Track an Event in Amplitude when a Recovery Key is Used During Password Reset

We currently on track when we send them an email confirming it was enabled.

Why measure this? It will help us better understand how we are at preventing people from being locked out of their accounts.

**Comment by irrationalagent** [comment]
Should this also be a user property? Say a boolean set to true when a user sets it up? Same with secondary email. We'd need to un-set it if they disable as well.

**Comment by davismtl** [comment]
Yes. It should be a user property too. That would be really valuable.

> We'd need to un-set it if they disable as well.
Correct, or if they use it.

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

In triage we thought this may already be done.  Do these exist in amplitude already?  Thanks

**Comment by irrationalagent** [comment]
Yes we should be able to close this now
